### ISSUE TITLE
[Enhancement]: add Filter "missing Series-Episode"

### ISSUE THREAD

**Issue by Gru0810** [advplyr/audiobookshelf#2850]
### Describe the feature/enhancement

It would be very helpful to have an additional filter for "missing episode items". I know that it's not possible to show the title but only to see which episode is missing can help to find the item and complete the serie.
In my case I have for example a serie "Commissario Montalbano":
![image](https://github.com/advplyr/audiobookshelf/assets/72630160/7c02d791-2de2-41f1-9e7a-68ba52da585b)
A list can look like this:
![image](https://github.com/advplyr/audiobookshelf/assets/72630160/464ecf23-dc43-4e25-820d-9492b13aefb3)
I only added a new table with Episode-numbers from 1 to 200 to be able to create a SQL-Statement to get all missing items based on the max number of episode-number.
A other solution can be to Add a field Episone_Count to the table "Series". But this field must be filled up manually.

**Comment by advplyr** [comment]
Is this along the same lines as https://github.com/advplyr/audiobookshelf/issues/963?

**Comment by Gru0810** [comment]
Yes more or less the same. But a first step it would be helpful to have a filter for "Missing episode-No" . To add the name of the missing item would be very cool but very hard to solve. For my germain-series often use https://www.lovelybooks.de. The sample serie above "Kommissar Montalbano" is presented by this : https://www.lovelybooks.de/autor/Andrea-Camilleri/reihe/Commissario-Montalbano-in-Reihenfolge-997773970/.
I manually copy-paste missing titles into my app:
![image](https://github.com/advplyr/audiobookshelf/assets/72630160/286ae2d5-0cbe-4660-961b-1176a2ae585d)
It can also be useful to a filed within table "series" like "series_URL" and perhaps Episode_count.

**Comment by Gru0810** [comment]
Yes, I agree with you. I didn't think about the decimal numbers. Another idea would be to have a section where admin-user can store SQL-Statements and execute them to get a user-defined list with the possibility to export the result as cvs or as Excel. So I would be able to get my list "Missing Series-Episode" . I know I need a table with incremental numbers within your SQLite DB. Is it allowed to create a Table by DDL-Script?

**Comment by mmuenker** [comment]
> I also don't see how this could be represented in the UI. The bookshelf are made up of actual items in the library so putting a block in there to represent a missing book wouldn't make sense.

*I am not familiar with the specifics of the data model, so it is possible that this solution may not be possible.*

I propose adding a new attribute `missing` that, when set to true, would visually disable (gray out) the item in the user interface.
